src/sdk/pynni/nni/compression/tensorflow/builtin_pruners.py

```diff
@@ -32,8 +32,8 @@ class AGP_Pruner(Pruner):
     """
     def __init__(self, config_list):
         """
-            Configure Args
-                initial_sparsity:
+            Configure Args:
```

Comment:
please follow the docstring format like other tuners?

Reply:
Updated.
Followed format used in `msg_dispatcher.handle_report_metric_data`

Reply:
I think this format has some differences with the format that we used(We used the format like xgboost).   I not sure about that. @leckie-chn

Reply:
This does not document the parameter list, instead it documents the fields of one parameter.